pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
did you change the executable bit? you'll need to commit and push that for it to stick otherwise every *nix user will have to do it themselves. thanks!